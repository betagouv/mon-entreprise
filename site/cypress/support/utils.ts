export const fr = Cypress.env('language') === 'fr'
export const baseUrl = Cypress.config('baseUrl')
