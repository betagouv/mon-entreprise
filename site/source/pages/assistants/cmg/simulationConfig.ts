import { SimulationConfig } from '@/domaine/SimulationConfig'

export const configCMG: SimulationConfig = {}
