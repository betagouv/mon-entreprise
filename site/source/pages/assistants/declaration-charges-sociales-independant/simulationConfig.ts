import { SimulationConfig } from '@/domaine/SimulationConfig'

export const config: SimulationConfig = {
	'objectifs exclusifs': [
		'déclaration charge sociales . résultat . cotisations obligatoires',
		'déclaration charge sociales . résultat . total charges sociales déductible',
		'déclaration charge sociales . résultat . revenu net fiscal',
		'déclaration charge sociales . résultat . assiette sociale',
	],
	situation: {
		'déclaration charge sociales': 'oui',
		'dirigeant . régime social': "'indépendant'",
		date: '01/01/2024',
		'dirigeant . indépendant . PL . CIPAV': 'non',
		'entreprise . catégorie juridique': "''",
		impôt: 'non',
	},
	'unité par défaut': '€',
}
