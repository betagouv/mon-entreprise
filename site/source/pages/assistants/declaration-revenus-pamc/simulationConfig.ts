import { SimulationConfig } from '@/domaine/SimulationConfig'

export const configDéclarationRevenusPAMC: SimulationConfig = {
	situation: {
		'déclaration revenus PAMC': 'oui',
	},
	'unité par défaut': '€',
}
