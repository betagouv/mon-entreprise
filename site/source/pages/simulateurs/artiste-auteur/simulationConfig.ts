import { SimulationConfig } from '../_configs/types'

export const configArtisteAuteur: SimulationConfig = {
	objectifs: [
		'artiste-auteur . cotisations',
		'artiste-auteur . revenus',
		'artiste-auteur . cotisations . IRCEC',
	],
	situation: {
		'artiste-auteur': 'oui',
		dirigeant: {
			'applicable si': 'non',
		},
	},
	'unité par défaut': '€/an',
	questions: {
		liste: [
			'artiste-auteur . revenus . BNC . micro-bnc',
			'artiste-auteur . cotisations . option surcotisation',
			'artiste-auteur . cotisations . IRCEC . cotisation RAAP . taux réduit',
			'artiste-auteur . cotisations . IRCEC . profession',
		],
	},
}
