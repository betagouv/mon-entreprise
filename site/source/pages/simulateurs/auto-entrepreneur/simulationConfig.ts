import { SimulationConfig } from '../_configs/types'

export const configAutoEntrepreneur: SimulationConfig = {
	'objectifs exclusifs': [
		"dirigeant . auto-entrepreneur . chiffre d'affaires",
		'dirigeant . auto-entrepreneur . revenu net',
		'dirigeant . auto-entrepreneur . revenu net . après impôt',
	],
	objectifs: [
		'dirigeant . auto-entrepreneur . cotisations et contributions',
		'dirigeant . rémunération . impôt',
	],
	questions: {
		"à l'affiche": [
			{
				label: "Type d'activité",
				dottedName: 'entreprise . activité . nature',
			},
			{
				label: 'Versement libératoire',
				dottedName:
					'dirigeant . auto-entrepreneur . impôt . versement libératoire',
			},
			{
				label: 'Impôt sur le revenu',
				dottedName: 'impôt . méthode de calcul',
			},
			{
				label: 'ACRE',
				dottedName: 'dirigeant . exonérations . ACRE',
			},
		],
		liste: [
			'entreprise',
			'dirigeant',
			'impôt',
			'établissement',
			'situation personnelle',
		],
		'non prioritaires': ['établissement . commune'],
		'liste noire': [
			'entreprise . activités',
			'entreprise . activités . revenus mixtes',
			'entreprise . charges',
			"entreprise . chiffre d'affaires",
		],
	},
	'unité par défaut': '€/an',
	situation: {
		salarié: 'non',
		'entreprise . catégorie juridique': "'EI'",
		'entreprise . catégorie juridique . EI . auto-entrepreneur': 'oui',
	},
}
