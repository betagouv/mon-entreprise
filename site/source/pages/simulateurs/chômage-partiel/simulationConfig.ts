import { SimulationConfig } from '../_configs/types'

export const configChômagePartiel: SimulationConfig = {
	objectifs: [
		'salarié . contrat . salaire brut',
		'salarié . rémunération . net . à payer avant impôt',
		'salarié . activité partielle . net habituel',
		'salarié . activité partielle . total employeur habituel',
		'salarié . coût total employeur',
	],
	questions: {
		liste: [
			'salarié . activité partielle',
			'salarié . temps de travail',
			'établissement . commune',
		],
	},
	'unité par défaut': '€/mois',
	situation: {
		dirigeant: 'non',
		'salarié . activité partielle': 'oui',
		'salarié . cotisations . prévoyances': {
			'applicable si': 'non',
		},
	},
}
