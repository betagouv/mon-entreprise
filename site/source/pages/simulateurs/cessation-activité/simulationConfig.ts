import { SimulationConfig } from '@/pages/simulateurs/_configs/types'
import { configIndépendant } from '@/pages/simulateurs/indépendant/simulationConfig'

export const configCessationActivité: SimulationConfig = {
	...configIndépendant,
	'unité par défaut': '€/an',
}
