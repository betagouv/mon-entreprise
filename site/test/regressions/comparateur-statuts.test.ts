import { it } from 'vitest'

import { configComparateurStatuts } from '@/pages/simulateurs/comparaison-statuts/simulationConfig'

import situations from './comparateur-statuts.yaml'
import { runSimulations } from './utils'

it('calculate simulations-salariÃ©', () => {
	runSimulations(
		situations,
		[
			...(configComparateurStatuts['objectifs exclusifs'] ?? []),
			...(configComparateurStatuts.objectifs ?? []),
		],
		configComparateurStatuts.situation
	)
})
