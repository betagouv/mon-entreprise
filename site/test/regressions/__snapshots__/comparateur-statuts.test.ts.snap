// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`calculate comparateur-statuts > ACRE 1`] = `
"dirigeant . rémunération . net: 44208
dirigeant . rémunération . net . après impôt: 44105
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: 9274
protection sociale . invalidité et décès . capital décès . orphelin: 193
protection sociale . invalidité et décès . pension de reversion: 392
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 487
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 725
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 24
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 64
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 1932
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 3864
protection sociale . retraite . base: 725
protection sociale . retraite . complémentaire: 65
protection sociale . retraite . trimestres: 4

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > ACRE 2`] = `
"dirigeant . rémunération . net: 37487
dirigeant . rémunération . net . après impôt: 32565
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: 9274
protection sociale . invalidité et décès . capital décès . orphelin: 193
protection sociale . invalidité et décès . pension de reversion: 873
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 970
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 1616
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 53
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 64
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 1932
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 3864
protection sociale . retraite . base: 1616
protection sociale . retraite . complémentaire: 145
protection sociale . retraite . trimestres: 4

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > ACRE 3`] = `
"dirigeant . rémunération . net: 31065
dirigeant . rémunération . net . après impôt: 28854
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: 1322
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: 826
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: 826
protection sociale . invalidité et décès . capital décès: 3681
protection sociale . invalidité et décès . capital décès . orphelin: null
protection sociale . invalidité et décès . pension de reversion: 892
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 991
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 1652
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: 51
protection sociale . maladie . arrêt maladie: 52
protection sociale . maladie . arrêt maladie . délai d'attente: 4
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 86
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: null
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: null
protection sociale . retraite . base: 1652
protection sociale . retraite . complémentaire: 159
protection sociale . retraite . trimestres: 4

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > EI à l'IS 1`] = `
"dirigeant . rémunération . net: 27503
dirigeant . rémunération . net . après impôt: 26070
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: 9274
protection sociale . invalidité et décès . capital décès . orphelin: 193
protection sociale . invalidité et décès . pension de reversion: 643
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 715
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 1191
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 39
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 64
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 1932
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 3864
protection sociale . retraite . base: 1191
protection sociale . retraite . complémentaire: 107
protection sociale . retraite . trimestres: 4

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > bas revenus 1`] = `
"dirigeant . rémunération . net: 1752
dirigeant . rémunération . net . après impôt: 1752
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: null
protection sociale . invalidité et décès . capital décès . orphelin: null
protection sociale . invalidité et décès . pension de reversion: 306
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 0
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 0
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 0
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: null
protection sociale . maladie . maternité paternité adoption: 6
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 193
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 386
protection sociale . retraite . base: 24
protection sociale . retraite . complémentaire: 2
protection sociale . retraite . trimestres: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > bas revenus 2`] = `
"dirigeant . rémunération . net: 577
dirigeant . rémunération . net . après impôt: 577
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: 9274
protection sociale . invalidité et décès . capital décès . orphelin: 193
protection sociale . invalidité et décès . pension de reversion: 306
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 487
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 686
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 25
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 64
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 1932
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 3864
protection sociale . retraite . base: 219
protection sociale . retraite . complémentaire: 2
protection sociale . retraite . trimestres: 3

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > bas revenus 3`] = `
"dirigeant . rémunération . net: 732
dirigeant . rémunération . net . après impôt: 732
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: 42
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: 26
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: 26
protection sociale . invalidité et décès . capital décès: 3681
protection sociale . invalidité et décès . capital décès . orphelin: null
protection sociale . invalidité et décès . pension de reversion: 306
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 31
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 52
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: 2
protection sociale . maladie . arrêt maladie: 0
protection sociale . maladie . arrêt maladie . délai d'attente: null
protection sociale . maladie . arrêt maladie . délai de carence: null
protection sociale . maladie . maternité paternité adoption: 0
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: null
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: null
protection sociale . retraite . base: 53
protection sociale . retraite . complémentaire: 5
protection sociale . retraite . trimestres: 0

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > base 1`] = `
"dirigeant . rémunération . net: 40548
dirigeant . rémunération . net . après impôt: 40445
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: 9274
protection sociale . invalidité et décès . capital décès . orphelin: 193
protection sociale . invalidité et décès . pension de reversion: 392
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 487
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 725
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 24
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 64
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 1932
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 3864
protection sociale . retraite . base: 725
protection sociale . retraite . complémentaire: 65
protection sociale . retraite . trimestres: 4

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > base 2`] = `
"dirigeant . rémunération . net: 32850
dirigeant . rémunération . net . après impôt: 29319
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: 9274
protection sociale . invalidité et décès . capital décès . orphelin: 193
protection sociale . invalidité et décès . pension de reversion: 768
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 854
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 1423
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 47
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 64
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 1932
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 3864
protection sociale . retraite . base: 1423
protection sociale . retraite . complémentaire: 127
protection sociale . retraite . trimestres: 4

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > base 3`] = `
"dirigeant . rémunération . net: 26920
dirigeant . rémunération . net . après impôt: 25474
entreprise . activité . nature . libérale . réglementée: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: 1147
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: 717
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: 717
protection sociale . invalidité et décès . capital décès: 3681
protection sociale . invalidité et décès . capital décès . orphelin: null
protection sociale . invalidité et décès . pension de reversion: 774
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 860
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 1433
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: 45
protection sociale . maladie . arrêt maladie: 47
protection sociale . maladie . arrêt maladie . délai d'attente: 5
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 74
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: null
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: null
protection sociale . retraite . base: 1434
protection sociale . retraite . complémentaire: 138
protection sociale . retraite . trimestres: 4

Notifications affichées : entreprise . TVA . franchise de TVA . notification"
`;

exports[`calculate comparateur-statuts > profession libérale non reglementée 1`] = `
"dirigeant . rémunération . net: 32220
dirigeant . rémunération . net . après impôt: 27054
entreprise . activité . nature . libérale . réglementée: false
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: 9274
protection sociale . invalidité et décès . capital décès . orphelin: 193
protection sociale . invalidité et décès . pension de reversion: 891
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 990
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 1650
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 54
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 64
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 1932
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 3864
protection sociale . retraite . base: 1650
protection sociale . retraite . complémentaire: 0
protection sociale . retraite . trimestres: 4"
`;

exports[`calculate comparateur-statuts > profession libérale non reglementée 2`] = `
"dirigeant . rémunération . net: 30852
dirigeant . rémunération . net . après impôt: 27944
entreprise . activité . nature . libérale . réglementée: false
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: null
protection sociale . invalidité et décès . capital décès: 9274
protection sociale . invalidité et décès . capital décès . orphelin: 193
protection sociale . invalidité et décès . pension de reversion: 721
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 802
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 1336
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: null
protection sociale . maladie . arrêt maladie: 44
protection sociale . maladie . arrêt maladie . délai d'attente: 12
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 64
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: 1932
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: 3864
protection sociale . retraite . base: 1336
protection sociale . retraite . complémentaire: 119
protection sociale . retraite . trimestres: 4"
`;

exports[`calculate comparateur-statuts > profession libérale non reglementée 3`] = `
"dirigeant . rémunération . net: 25218
dirigeant . rémunération . net . après impôt: 24030
entreprise . activité . nature . libérale . réglementée: false
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente décès: 1075
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: 672
protection sociale . invalidité et décès . accidents du travail et maladies professionnelles . rente incapacité: 672
protection sociale . invalidité et décès . capital décès: 3681
protection sociale . invalidité et décès . capital décès . orphelin: null
protection sociale . invalidité et décès . pension de reversion: 726
protection sociale . invalidité et décès . pension invalidité . invalidité partielle: 806
protection sociale . invalidité et décès . pension invalidité . invalidité totale: 1344
protection sociale . maladie . accidents du travail et maladies professionnelles . indemmnités: 42
protection sociale . maladie . arrêt maladie: 44
protection sociale . maladie . arrêt maladie . délai d'attente: 5
protection sociale . maladie . arrêt maladie . délai de carence: 3
protection sociale . maladie . maternité paternité adoption: 70
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos adoption: null
protection sociale . maladie . maternité paternité adoption . allocation forfaitaire de repos maternel: null
protection sociale . retraite . base: 1344
protection sociale . retraite . complémentaire: 129
protection sociale . retraite . trimestres: 4"
`;
